pilot/pkg/xds/mesh_network_test.go

```diff
@@ -227,7 +227,7 @@ func TestMeshNetworking(t *testing.T) {
 						Labels: map[string]string{
 							label.TopologyNetwork.Name: "network-2",
 							// set the label here to test it = expectation doesn't change since we map back to that via NodePort
-							controller.IstioGatewayPortLabel: "443",
+							annotation.NetworkingGatewayPort.Name: "443",
```

Comment:
why is it "annotation" if its in a label? can be it be label or annotation?

Reply:
`IstioGatewayPortLabel` is moved into the `annotation` package
https://github.com/istio/api/blob/5796da5cb825caed340bab1c7eacd8290b625f35/annotation/annotations.gen.go#L269

Reply:
do I need to move `IstioGatewayPortLabel` to the `label` package? 
i can do it